### DISCUSSION TITLE
Permissions for listing repository events

### DISCUSSION THREAD

**Discussion by filipstefansson** [community/community#24417] [Apps, API and Webhooks]
<p>I’m trying to use the following endpoint: <a href="https://developer.github.com/v3/activity/events/#list-repository-events" rel="nofollow noopener">https://developer.github.com/v3/activity/events/#list-repository-events</a>, but I can’t figure out how to get a response. I get Resource not accessible by integration. </p>
<p>I can’t find what permissions I need to add to the app to be able to get a response. Is this documented anywhere? I’ve tried authenticating as the user, the app and as the installation, but no luck.</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>It appears that endpoint isn’t available to GitHub Apps. If it were, you would see the info icon after the title:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/6/6db32696887d15dc2a4c92674b1485b0a3ec5a4b.png" data-download-href="/uploads/short-url/fErS7EaBby2FZgOLcfZxvhTwdPJ.png?dl=1" title="Screen Shot 2019-06-03 at 5.17.43 PM.png"><img src="https://user-images.githubusercontent.com/1038121/181084680-e394b60a-b08c-4673-885c-5beefcf02711.png" alt="Screen Shot 2019-06-03 at 5.17.43 PM.png" data-base62-sha1="fErS7EaBby2FZgOLcfZxvhTwdPJ" width="690" height="147" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/6/6db32696887d15dc2a4c92674b1485b0a3ec5a4b_2_690x147.png, https://aws1.discourse-cdn.com/github/optimized/2X/6/6db32696887d15dc2a4c92674b1485b0a3ec5a4b_2_1035x220.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/6/6db32696887d15dc2a4c92674b1485b0a3ec5a4b_2_1380x294.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/6/6db32696887d15dc2a4c92674b1485b0a3ec5a4b_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2019-06-03 at 5.17.43 PM.png</span><span class="informations">1382×296 19.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>The list of <a href="https://developer.github.com/v3/apps/available-endpoints/#events">events endpoints that can be accessed by GitHub Apps</a> is available in the documentation.</p>
<p>I hope that helps!</p>
